src/OAuth/AccessKey.cs

```diff
@@ -20,5 +22,14 @@ public class AccessKey
 
         [JsonProperty("jwk")]
         public JsonWebKey Jwk { set; get; }
+
+        private static AccessKey accessKey;
+
+        public static AccessKey DecodeBase64(string encoded)
```

Comment:
Should this be named the same as the Java equivalent function? `CreateFromBase64EncodedAccessKey`